src/main/java/com/programmers/springbootbasic/common/handler/CommandHandler.java

```diff
@@ -0,0 +1,214 @@
+package com.programmers.springbootbasic.common.handler;
+
+import com.programmers.springbootbasic.common.response.model.CommonResult;
+import com.programmers.springbootbasic.common.utils.ConsoleIOManager;
+import com.programmers.springbootbasic.domain.customer.controller.CustomerController;
+import com.programmers.springbootbasic.domain.voucher.controller.VoucherController;
+import com.programmers.springbootbasic.domain.wallet.controller.WalletController;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Controller;
+
+import java.io.IOException;
+
+@Slf4j
+@Controller
+@RequiredArgsConstructor
+public class CommandHandler {
+    private final ConsoleIOManager consoleIOManager;
+    private final VoucherController voucherController;
+    private final CustomerController customerController;
+    private final WalletController walletController;
+
+    public void run() {
+        CommandType command;
+        consoleIOManager.printProgramSelectMenu();
```

Comment:
현재 프로그램 선택 메뉴를 출력하기 위해 사용되는 메서드로 보입니다.

이런 경우 상황마다 출력을 위해 메서드를 만들어야 한다는 문제가 발생할 것 같습니다.

출력시에 하나의 메서드만 이용하도록 만드는 것에 대해 어떻게 생각하시나요?

Reply:
설계 과정에서 프로그램 메뉴 출력 메소드를 나눈 이유는 이후 프로그램 수정이나 추가 과정에서 용이할 것이라고 판단하여 이러한 방식으로 작성을 했습니다. 하지만 멘토님 리뷰를 보고 생각해보니 출력에 대한 메서드를 각각 따로 만드는 것이 코드의 길이나 난잡함을 일으킬 수 있지 않을까? 라는 생각이 들었습니다.
```suggestion
        consoleIOManager.println(ENUM_VALUES);
```
위와 같은 방식으로 이미 존재하는 println 메소드를 활용해서 Enum으로 출력 값만 따로 관리하는 형식이 좋을것 같습니다!